packages/react-app/src/components/BuilderCard.jsx

```diff
@@ -1,14 +1,13 @@
 import React from "react";
+import { Link } from "react-router-dom";
 import Address from "./Address";
 
 const BuilderCard = ({ builder, mainnetProvider }) => (
-  // ToDo. Link to Builder profile.
-  <div style={{ display: "inline-block" }}>
-    <Address
-      address={builder.id}
-      ensProvider={mainnetProvider}
-    />
-  </div>
+  <Link to={`/builders/${builder.id}`}>
+    <div style={{ display: "inline-block" }}>
+      <Address address={builder.id} ensProvider={mainnetProvider} />
+    </div>
+  </Link>
```

Comment:
```suggestion
  <Link to={`/builders/${builder.id}`} style={{ display: "inline-block" }}>
    <Address address={builder.id} ensProvider={mainnetProvider} />
  </Link>
```
Have you tried this? I'm not 100% sure if Link passes the `style` prop to their internal components, but if it did, the `<div>` wouldn't be necessary anymore.